worklist/views.py

```diff
@@ -59,11 +101,17 @@ def create_worklist(request):
 
         return JsonResponse(content)
     logger.info('Message:' + str(content['message']))
-    return render(request, 'create-worklist.html', {'content': content})
+    return render(request, 'create-worklist.html', {'content': content,
```

Comment:
When we succeed in creating the worklist, should this redirect to the worklist page itself (show-tasks), rather than back to the form?

Reply:
Yeah, I think I prefer show-tasks. Staying on the same form was pretty surprising when I tried it, and I imagine that users are more likely to want to see the worklist they've just created than to batch-create worklists.